pkg/provision/storage/asyncStorage.go

```diff
@@ -121,7 +123,7 @@ func (p *AsyncStorageProvisioner) ProvisionStorage(podAdditions *v1alpha1.PodAdd
 	deploy, err := asyncstorage.SyncWorkspaceSyncDeploymentToCluster(workspace, configmap, pvcName, clusterAPI)
 	if err != nil {
 		if errors.Is(err, asyncstorage.NotReadyError) {
-			return &NotReadyError{
+			return &dwerrors.RetryError{
 				Message:      "waiting for async storage server deployment to be ready",
```

Comment:
Extreme nitpick (that isin't worth immediately addressing IMO..): Some error messages [are now capitalized](https://github.com/devfile/devworkspace-operator/pull/1064/files#diff-bfb432d51dee16f88d16a491fd7b8f24fbe2d56bc53d1036fee90e6fa27e0354R88) while others are not. I imagine the new standard is that errors should start with a capital letter.

When this gets merged, I can make an issue regarding this (i.e. that all DW error messages should start with a capital letter).